src/System.Private.Xml/src/System/Xml/BinaryXml/SqlUtils.cs

```diff
@@ -626,7 +635,7 @@ internal abstract class BinXmlDateTime
     {
         private const int MaxFractionDigits = 7;
 
-        internal static int[] KatmaiTimeScaleMultiplicator = new int[8] {
+        static internal int[] KatmaiTimeScaleMultiplicator = new int[8] {
```

Comment:
Actually, I think I was wrong. I checked CodeFormatter with a small csproj and it seems it doesn't have a preference over the order of this kind. This must be a change I accidentally made myself. Will fix.